crud/stats/local_registry.lua

```diff
@@ -0,0 +1,124 @@
+local dev_checks = require('crud.common.dev_checks')
+local registry_common = require('crud.stats.registry_common')
+
+local registry = {}
+local internal_registry = {}
+
+--- Initialize local metrics registry
+--
+--  Registries are not meant to used explicitly
+--  by users, init is not guaranteed to be idempotent.
+--
+-- @function init
+--
+-- @treturn boolean Returns true.
+--
+function registry.init()
+    internal_registry.spaces = {}
+    internal_registry.space_not_found = 0
+
+    return true
+end
+
+--- Destroy local metrics registry
+--
+--  Registries are not meant to used explicitly
+--  by users, destroy is not guaranteed to be idempotent.
+--
+-- @function destroy
+--
+-- @treturn boolean Returns true.
+--
+function registry.destroy()
+    internal_registry = {}
+
+    return true
+end
+
+--- Get copy of local metrics registry
+--
+--  Registries are not meant to used explicitly
+--  by users, get is not guaranteed to work without init.
+--
+-- @function get
+--
+-- @tparam string space_name
+--  (Optional) If specified, returns table with statistics
+--  of operations on table, separated by operation type and
+--  execution status. If there wasn't any requests for table,
+--  returns {}. In not specified, returns table with statistics
+--  about all existing spaces and count of calls to spaces
+--  that wasn't found.
+--
+-- @treturn table Returns copy of metrics registry (or registry section).
+--
+function registry.get(space_name)
```

Comment:
Well, I understand if it would be convenient to support mapping space id to name in external 'stats.get", but I am against supporting it in internal storage level.

Reply:
In case you asking why we need to map id to name: if `get(space_id)` and `get(space_name)` will store (and return) different stats, it would be awfully confusing and incorrect.

Reply:
space_id for different instances could be different. It's better to use space name as identifier

Reply:
Now we map space_id for `crud.len` requests based on router schema. Does that mean `crud.len` may not work as expected?

Reply:
Try to create local/temporary space on several instances. After you even apply "common" schema for all replicasets, you will have different space_id for spaces with equal names.

In case of cartidge you can have two replicasets with roles: {vshard-storage + user_storage} and vshard-storage.
In case if user_storage creates spaces and is initialized before storage you will have different schema on storages. Of course, using numerical space id for sharded system is a wrong way.

Reply:
Referred in a ticket: https://github.com/tarantool/crud/issues/255